frontend/src/components/forms/CreateEmployee.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useEffect, useState } from "react";
+import {
+  Box,
+  Button,
+  Checkbox,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Radio,
+  RadioGroup,
+  Text,
+} from "@chakra-ui/react";
+import { AddIcon } from "@chakra-ui/icons";
+import commonApiClient from "../../APIClients/CommonAPIClient";
+import RoleOptions from "../common/types/Roles";
+import { INVITE_EMPLOYEE_ERROR } from "../../constants/ErrorMessages";
+import CreateToast from "../common/Toasts";
+
+const CreateEmployee = (): React.ReactElement => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [isDisabled, setIsDisabled] = useState<boolean>(false);
+  const newToast = CreateToast();
+
+  const [invitedEmail, setInvitedEmail] = useState<string>("");
+  const [invitedFirstName, setInvitedFirstName] = useState<string>("");
+  const [invitedLastName, setInvitedLastName] = useState<string>("");
+  const [invitedAdminStatus, setinvitedAdminStatus] = useState<string>("");
+  const [
+    isTwoFactorAuthenticated,
+    setIsTwoFactorAuthenticated,
+  ] = useState<boolean>(false);
+
+  const [invitedEmailError, setInvitedEmailError] = useState<boolean>(false);
+  const [invitedFirstNameError, setInvitedFirstNameError] = useState<boolean>(
+    false,
+  );
+  const [invitedLastNameError, setInvitedLastNameError] = useState<boolean>(
+    false,
+  );
+  const [
+    invitedAdminStatusError,
+    setInvitedAdminStatusError,
+  ] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (invitedAdminStatus === "1") {
+      setIsDisabled(true);
+    } else {
+      setIsDisabled(false);
+    }
+  }, [invitedAdminStatus]);
+
+  const handleFirstNameChange = (e: { target: { value: unknown } }) => {
```

Comment:
Currently users can type numbers into the name input boxes. You can use regex to block them from putting anything other than letters:
```
const handleFirstNameChange = (e: { target: { value: unknown } }) => {
    const inputValue = e.target.value as string;
    if (/^[a-z]{0,}$/i.test(inputValue)) {
      setInvitedFirstName(inputValue);
      setInvitedFirstNameError(false);
    }
  };
```

Also, since there is error validation on submit you don't need `setInvitedFirstNameError(true);`